---
locations:
  - display_name: Body Behavior
    package: bitbots_body_behavior
    debug_topic: debug/dsd/body_behavior
    relative_action_path: actions
    relative_decision_path: decisions
    relative_dsd_path: main.dsd
  - display_name: Head Behavior
    package: bitbots_head_behavior
    debug_topic: debug/dsd/head_behavior
    relative_action_path: actions
    relative_decision_path: decisions
    relative_dsd_path: head_behavior.dsd
  - display_name: HCM
    package: bitbots_hcm
    debug_topic: debug/dsd/hcm
    relative_action_path: actions
    relative_decision_path: decisions
    relative_dsd_path: hcm.dsd
  - display_name: Localization
    package: bitbots_localization_handler
    debug_topic: debug/dsd/localization
    relative_action_path: actions
    relative_decision_path: decisions
    relative_dsd_path: localization.dsd
